WooCommerce/Classes/ViewRelated/Reviews/ReviewsViewModel.swift

```diff
@@ -31,6 +64,10 @@ final class ReviewsViewModel {
         return data.notifications.filter { $0.read == false }
     }
 
+    var shouldPromptForAppReview: Bool {
```

Comment:
Nit: Let's also add a comment describing this property.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/5834/commits/c0e8dfca3fd09fc6395f097564342108496f5c84